xformers/ops/swiglu_op.py

```diff
@@ -188,7 +188,7 @@ class _ForwardToPythonAutogradFunc(SwiGLUOp):
     def supports(self, op: "SwiGLUOpDispatch") -> bool:
         # Let's disable autocast in bf16 until this issue is fixed
         # https://github.com/pytorch/pytorch/issues/87979
-        if op.dtype_autocast_gpu == torch.bfloat16:
+        if op.dtype_autocast_gpu == torch.bfloat16 and torch.__version__ < "2.0.0":
             return False
```

Comment:
I believe we can remove this check all together, because xFormers now requires pytorch 2.2+
https://github.com/facebookresearch/xformers/blob/67f38b9dffc7696a6731fe3f3964369ec8113bb4/requirements.txt#L3

Reply:
Yeah, you're right. It can be removed. I just simply removed this check haha